corehq/apps/api/tests.py

```diff
@@ -2027,23 +2027,45 @@ def _get_query_params(url):
             self.domain.name, "123", 100, 50, 120, query_dict)
         self.assertEqual(next, "")
 
-    def test_auth(self):
-        user_in_wrong_domain_name = 'Mallory'
+    def test_auth_capital_username(self):
+        capital_username_credentials = self._get_basic_credentials(self.username.upper(), self.password)
+        response = self.client.get(
+            self.single_endpoint(self.report_configuration._id),
+            HTTP_AUTHORIZATION='Basic ' + capital_username_credentials
+        )
+        self.assertEqual(response.status_code, 200)
```

Comment:
nit: these tests all have basically the same last 5 lines - might be nice to swap them with something like
```python
self.assertUserGetsResponse(credentials, 200)
```